praw/models/stylesheet.py

```diff
@@ -21,3 +24,18 @@ class Stylesheet(PRAWBase):
     ``stylesheet``          The contents of the stylesheet, as CSS.
     ======================= ===================================================
     """
+
+    def __eq__(self, other: _Stylesheet) -> bool:
+        """Check if two stylesheets are equal."""
+        if isinstance(other, self.__class__):
+            return (
+                self.images == other.images
+                and self.stylesheet == other.stylesheet
+            )
+        return super().__eq__(other)
+
+    def __repr__(self):
+        """Return the object's REPR."""
+        return "<{} with {} characters and {} images>".format(
```

Comment:
This repr doesn't follow (what I see as) the established convention for PRAW's reprs, which is that they indicate information that can be used to reconstruct the instance. I'm basing this understanding largely on [this comment from @bboe](https://github.com/praw-dev/praw/pull/942#discussion_r189743828). I would be in favor of just not having a `__repr__` for the `Stylesheet` class.

See also https://docs.python.org/3/library/functions.html#repr